[PL-51473] PL 1.39.x Release Notes (#6831)
harness/developer-hub: release-notes/platform.md

title: Platform release notes
sidebar_label: Platform
tags: [NextGen, "platform"]

> date: 2024-05-27:T10:00:30

Reviewer: ```suggestion
date: 2024-05-28:T10:00:30
```

## May 2024

> ### Version 1.39.x<!--  May 27, 2024 -->

Reviewer: ```suggestion
### Version 1.39.4<!--  May 27, 2024 -->
```

#### Fixed issues

> - Keyboard navigation was not functioning for the User Select dropdown in the UserGroup form, and uncontrolled form submission occurred when pressing Enter. Updated the User Select dropdown to support keyboard navigation. Additionally, the form now properly handles uncontrolled submissions when the Enter key is pressed, allowing users to select items using the keyboard without unintended form submissions. (PL-51168, ZD-62169)

Reviewer: @saifeemustafaqharness I suggest watching the video in the Jira. The dropdown name in the UI is **Add Users**, not User Select. The issue occurred when adding new members to a user group.

> - Keyboard navigation was not functioning for the User Select dropdown in the UserGroup form, and uncontrolled form submission occurred when pressing Enter. Updated the User Select dropdown to support keyboard navigation. Additionally, the form now properly handles uncontrolled submissions when the Enter key is pressed, allowing users to select items using the keyboard without unintended form submissions. (PL-51168, ZD-62169)

Reviewer: ```suggestion
- Keyboard navigation was not functioning for the **Add Users** list when adding a new member to a user group, and uncontrolled form submission occurred when pressing Enter. Updated the **Add Users** list to support keyboard navigation. Additionally, the form now properly handles uncontrolled submissions when the Enter key is pressed, allowing users to select items using the keyboard without unintended form submissions. (PL-51168, ZD-62169)
```

> - Users could be added to SCIM Provisioned/Externally Managed User Groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and the addition of users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)

Reviewer: ```suggestion
- Users could be added to SCIM-provisioned/externally managed user groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and the addition of users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)
```

> - Users could be added to SCIM-provisioned/externally managed user groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and the addition of users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)

Reviewer: ```suggestion
- Users could be added to SCIM-provisioned/externally managed user groups from the Harness UI, even though membership edits for externally managed groups were not allowed. The issue has been fixed, and adding users to externally managed user groups via the Harness UI is no longer supported. (PL-50663)
```

## May 2024

> ### Version 1.39.4<!--  May 27, 2024 -->

Reviewer: ```suggestion
### Version 1.39.4<!--  May 28, 2024 -->
```
